Cargo.toml

```diff
@@ -28,6 +28,7 @@ netlink-proto = { default-features = false, version = "0.11" }
 nix = { version = "0.26.1", default-features = false, features = ["fs", "mount", "sched", "signal"] }
 tokio = { version = "1.0.1", features = ["rt"], optional = true}
 async-global-executor = { version = "2.0.2", optional = true }
+macaddr = "1.0"
```

Comment:
not sure we want to consume this dependency.

Reply:
Makes sense. 

As a follow up I can try to port (new implementation) the required functions into the project, thus removing the need for the dependency.